elements/trove-mysql-guest-configuration/post-install.d/63-mysql

```diff
@@ -88,7 +88,7 @@ swift_service_type = $DIB_TROVE_SWIFT_SERVICE_TYPE
 # ========== Datastore Manager Configurations ==========
 # Datastore manager implementations.
 # Format: list of 'datastore-type:datastore.manager.implementation.module'
-# datastore_registry_ext = mysql:trove.guestagent.datastore.mysql.manager.Manager, percona:trove.guestagent.datastore.mysql.manager.Manager
+datastore_registry_ext = mysql:trove.guestagent.datastore.mysql.manager.Manager, percona:trove.guestagent.datastore.mysql.manager.Manager
```

Comment:
Setting this brings the following error: 

`Mar 08 11:12:39 db2.novalocal systemd[1]: Started OpenStack Trove Guest.
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: Traceback (most recent call last):
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: File "/usr/bin/trove-guestagent", line 10, in <module>
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: sys.exit(main())
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: File "/usr/lib/python2.7/site-packages/trove/cmd/guest.py", line 37, in main
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: cfg.parse_args(sys.argv)
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: File "/usr/lib/python2.7/site-packages/trove/common/cfg.py", line 1578, in parse_args
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: default_config_files=default_config_files)
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: File "/usr/lib/python2.7/site-packages/oslo_config/cfg.py", line 2575, in __call__
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: else sys.argv[1:])
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: File "/usr/lib/python2.7/site-packages/oslo_config/cfg.py", line 3316, in _parse_cli_opts
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: return self._parse_config_files()
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: File "/usr/lib/python2.7/site-packages/oslo_config/cfg.py", line 3352, in _parse_config_files
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: self._oparser.parse_args(self._args, namespace)
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: File "/usr/lib/python2.7/site-packages/oslo_config/cfg.py", line 2366, in parse_args
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: return super(_CachedArgumentParser, self).parse_args(args, namespace)
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: File "/usr/lib64/python2.7/argparse.py", line 1688, in parse_args
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: args, argv = self.parse_known_args(args, namespace)
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: File "/usr/lib64/python2.7/argparse.py", line 1720, in parse_known_args
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: namespace, args = self._parse_known_args(args, namespace)
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: File "/usr/lib64/python2.7/argparse.py", line 1926, in _parse_known_args
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: start_index = consume_optional(start_index)
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: File "/usr/lib64/python2.7/argparse.py", line 1866, in consume_optional
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: take_action(action, args, option_string)
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: File "/usr/lib64/python2.7/argparse.py", line 1794, in take_action
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: action(self, namespace, argument_values, option_string)
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: File "/usr/lib/python2.7/site-packages/oslo_config/cfg.py", line 1803, in __call__
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: ConfigParser._parse_file(values, namespace)
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: File "/usr/lib/python2.7/site-packages/oslo_config/cfg.py", line 2058, in _parse_file
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: raise ConfigFileParseError(pe.filename, str(pe))
Mar 08 11:14:31 db2.novalocal trove-guestagent[3040]: oslo_config.cfg.ConfigFileParseError: Failed to parse /etc/trove/trove-guestagent.conf: at /etc/trove/trove-guestagent.conf:70, Unexpected continuation line: ' datastore_registry_ext = mysql:trove.guestagent.datastore.mysql.manager.Manager, percona:trove.guestagent.datastore.mysql.manager.Manager'
Mar 08 11:14:32 db2.novalocal systemd[1]: openstack-trove-guestagent.service: main process exited, code=exited, status=1/FAILURE
Mar 08 11:14:32 db2.novalocal systemd[1]: Unit openstack-trove-guestagent.service entered failed state.
Mar 08 11:14:32 db2.novalocal systemd[1]: openstack-trove-guestagent.service failed.
`